src/server/game/Spells/Spell.cpp

```diff
@@ -3814,6 +3806,17 @@ void Spell::update(uint32 difftime)
                         if (Unit* unit = m_caster->GetGUID() == target.TargetGUID ? m_caster->ToUnit() : ObjectAccessor::GetUnit(*m_caster, target.TargetGUID))
                             unit->RemoveOwnedAura(m_spellInfo->Id, m_originalCasterGUID, 0, AURA_REMOVE_BY_CANCEL);
                 }
+                else
+                {
+                    // Update serverside orientation of channeling units
+                    if (Unit* unitCaster = m_caster->ToUnit())
+                    {
+                        ObjectGuid const channelGuid = unitCaster->GetChannelObjectGuid();
+                        if (!channelGuid.IsEmpty() && channelGuid != unitCaster->GetGUID() && GetSpellInfo()->HasAttribute(SPELL_ATTR1_CHANNEL_TRACK_TARGET))
+                            if (WorldObject const* objectTarget = ObjectAccessor::GetWorldObject(*unitCaster, channelGuid))
```

Comment:
do we truly need this on every tick?

is it not sufficient to do it whenever the channel would "do" anything (trigger a spell, periodic tick on an aura, etc.)?

are there any channels that truly need _constant_ facing updates?

Reply:
Yes. channeled spells that periodically trigger a spell that for example cone target spells that constantly need a up to date angle

Reply:
sure, but couldn't we just update the facing when the channel aura triggers a spell?

i.e. before processing any periodic event of the channel aura, update facing

Reply:
I think these comments should be considered, maybe this has broken things: https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/commit/d606fa4688c01652948be36097014c029b43f1a4?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDExODI3MTMxNzI6NTIxMDM2